app/lumina.js

```diff
@@ -0,0 +1,25 @@
+import { useRef, useState, useEffect, createContext } from "react";
+
+import { spawnNode } from "lumina-node";
+
+export const LuminaContext = createContext(null);
+
+export function LuminaContextProvider({ children }) {
+  const [lumina, setLumina] = useState(null);
+  const initialized = useRef(false);
```

Comment:
can we actually switch to useState here? It's not wrong i think, but I used it initially because I confused `useEffect` and regular callbacks

Reply:
same [as here](https://github.com/eigerco/lumina-front/pull/55#discussion_r1775208985), this causes double worker init, whatever I try.